sensing/pointcloud_preprocessor/README.md

```diff
@@ -45,14 +45,23 @@ Detail description of each filter's algorithm is in the following links.
 
 ### Node Parameters
 
-| Name               | Type   | Default Value | Description                           |
-| ------------------ | ------ | ------------- | ------------------------------------- |
-| `input_frame`      | string | " "           | input frame id                        |
-| `output_frame`     | string | " "           | output frame id                       |
-| `max_queue_size`   | int    | 5             | max queue size of input/output topics |
-| `use_indices`      | bool   | false         | flag to use pointcloud indices        |
-| `latched_indices`  | bool   | false         | flag to latch pointcloud indices      |
-| `approximate_sync` | bool   | false         | flag to use approximate sync option   |
+{{ json_to_markdown("sensing/pointcloud_preprocessor/schema/filter.schema.json") }}
+
+### blockage_diag
```

Comment:
These should not be here.
There are documentations for each node, please put them in those documentations.

Reply:
Got, already put them in doc.